site/_posts/2020-10-06-release-1.26.0.md

```diff
@@ -30,6 +30,10 @@ The [Apache Calcite PMC]({{ site.baseurl }})
 is pleased to announce
 [Apache Calcite release 1.26.0]({{ site.baseurl }}/docs/history.html#v1-26-0).
 
+**Warning:** Calcite 1.26.0 has **severe** issues with `RexNode` simplification caused by `SEARCH operator` (
+wrong data from query optimization, `NullPointerException`), so use 1.26.0 for development only,
+and beware that Calcite 1.26.0 would corrupt your data.
+
 This release comes about two months after 1.25.0 and includes more than 70 resolved
```

Comment:
Would be better if you add the related jira id in the message?

Reply:
I've added references